src/types/inventory.ts

```diff
@@ -122,3 +122,7 @@ export interface CreatedProduct {
 export interface CreatedProductTypes {
   productTypes: ProductTypeInterface;
 }
+
+export interface DeleteProduct {
+  id: boolean;
```

Comment:
Question: Why the id is a boolean?

Reply:
It's a mistake, it should be 'deleteProduct' instead of 'id'